ovn-fake-multinode-utils/process-monitor.py

```diff
@@ -36,7 +36,14 @@ def monitor(suffix: str, out_file: str, exit_file: str) -> None:
             tme = time.time()
             for p in processes:
                 try:
-                    name = p.name() + "-" + suffix + "-" + str(p.pid)
+                    name = p.name()
+                    for arg in p.cmdline():
+                        if arg.endswith('.pid') or arg.endswith('.py'):
+                            name = arg.split('/')[-1].split('.')[0]
+                            break
```

Comment:
Is there a legitimate case where there could be multiple `/` or `.` characters in the file name? If so, this could yield odd results. I tend to use Pathlib methods for getting what I want from a pathname. In this case, you can do
```
    # Assuming you have "from pathlib import PurePosixPath" in this file
    arg_path = PurePosixPath(arg)
    if arg_path.suffix == '.pid' or arg_path.suffix == '.py':
        name = arg_path.stem
        break
```
It's completely up to you if you think it's worth doing this way instead of the `split` way.

Reply:
I've just now noticed the `[-1]` in your method, so you are accounting for multiple `/` characters. Sorry about that.

Reply:
No problem.  :)
In fact, we almost always have more than 1 `/`.  Current version seems to work fine with that.